.github/workflows/ci.yaml

```diff
@@ -20,6 +20,9 @@ permissions:
   id-token: write
   contents: write
 
+env:
+  DEBUG: ${{ secrets.ACTIONS_STEP_DEBUG }}
```

Comment:
Was this just part of testing or do we still need this?

Reply:
I _think_ `secrets.ACTIONS_STEP_DEBUG` is what GitHub sets when you click the "run with debugging" option ([docs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging#enabling-step-debug-logging)).

Since our go e2e tests can't (easily) read the secret, I just propagate it to the environment. Forgot to update the change list in the description of the PR to include this